Makefile

```diff
@@ -7,6 +7,13 @@ SUPERSET_DIR = superset
 # Directory with custom code to copy into SUPERSET_DIR
 PATCH_SOURCE_DIR = srcd
 
+# Directory with template for docker-compose.override.yml
+OVERRIDE_TEMPLATE_PATH := $(PATCH_SOURCE_DIR)/contrib/docker/docker-compose.override.yml
+# Directory for the sourced global docker-compose.override.yml
+OVERRIDE_OUTPUT_PATH := $(HOME)/.sourced/compose-files/__active__/docker-compose.override.yml
+# Directory for the docker-compose.override.yml backup
+OVERRIDE_BACKUP_PATH := $(OVERRIDE_OUTPUT_PATH).bak
```

Comment:
what is the use case for backup? do you handle the situation when `set-override` was called twice?

Reply:
Yes, if you call `set-override` twice, `$(OVERRIDE_BACKUP_PATH)` target is called only once, because it's a regular one (not a `.PHONY` one) (see [`Makefile::68`](https://github.com/src-d/sourced-ui/pull/221/files#diff-b67911656ef5d18c4ae36cb6741b7965R68))